overlord/state/taskrunner.go

```diff
@@ -46,6 +46,17 @@ func (r *Retry) Error() string {
 	return "task should be retried"
 }
 
+// Hold is returned from a handler to signal that the task cannot
+// proceed at the moment and that some manual action from the user
+// is usually required at this point.
```

Comment:
this is probably ok but we need to think a bit terminology or comments because so far HoldStatus was meant to thing that could not happen because errors. See the doc comment for it.